bin/slurmjob.sh

```diff
@@ -4,7 +4,7 @@
 #SBATCH --reservation=clima
 
 BUILDKITE_PATH=${BUILDKITE_PATH:=/groups/esm/slurm-buildkite}
-BUILDKITE_QUEUE=${BUILDKITE_QUEUE:=central}
+BUILDKITE_QUEUE=${BUILDKITE_QUEUE:=new-central}
```

Comment:
@Sbozzolo, so is this one

Reply:
(and below in this file)

Reply:
BUILDKITE_QUEUE is set in the cron.sh job (and has to be set carefully). The one below is just a comment. We can update them, but they don't affect anything.